numpy/f2py/_isofenv.py

```diff
@@ -0,0 +1,23 @@
+"""
+ISO_FORTRAN_ENV maps for f2py2e
+"""
+iso_fortran_env_map = {
+    'integer': {
+        'int8': 'signed_char',
+        'int16': 'short',
+        'int32': 'int',
+        'int64': 'long_long',
+    },
+    'real': {
+        'real32': 'float',
+        'real64': 'double',
+    }
+}
+
+isof_f2pycode_map = {}
+iso_c2py_map = {}
+
+isof_kindmap = {}
+for fortran_type, std_type_dict in iso_fortran_env_map.items():
+    for std_type in std_type_dict.keys():
+        isof_kindmap[std_type] = fortran_type
```

Comment:
These local variables will cause some namespace pollution. And because there's no `__all__`, stubtest is probably going to require stub annotations for e.g. `std_type`.  

You can see some examples of that in #28522 in the places I marked with `# namespace pollution`. So it'd help we can prevent it this time :)